tools/localization_analysis/scripts/make_groundtruth.py

```diff
@@ -41,24 +41,19 @@
         help="Existing map to use as basis for groundtruth.  Should largely overlap area covered in input bagfile.",
     )
     parser.add_argument(
-        "maps_directory",
-        help="Location of images used for each bagfile use to generate base_surf_map.",
+        "--loc-map",
+        default="",
+        help="Full path to Localization map for bagfile to test localization performance.",
```

Comment:
Add that if no loc map is passed, the localization test is not run

Reply:
added that comment